modules/ob-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assemblies:
+// * builds/ob-release-notes.adoc
+
+[id="compatibility-support-matrix_{context}"]
+= Compatibility and support matrix
+
+// Some features in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features are not intended for production use.
+
+In the table, components are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+The link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features are experimental features and are not intended for production use.
+
+.Compatibility and support matrix
+[options="header"]
+|===
+
+| {builds-title} Version 2+| Component Version | Compatible
+Openshift Pipelines Version | OpenShift Version
+
+| Operator | Builds (Shipwright) | CLI | |
```

Comment:
I'm not sure if a formatting issue is evident here. It looks like each column has two headings (for example: Red Hat OpenShift Builds Version and Operator). 

Maybe merge two row into one? For example, `Operator 1.0`.

Reply:
This table is finalized by PM. So, do not want to change it.